testapps/on_device_unit_tests/setup.py

```diff
@@ -39,7 +40,7 @@
     'apk':
         {
             'requirements':
-                'sqlite3,libffi,openssl,pyjnius,kivy,python3,requests',
+                'sqlite3,libffi,openssl,pyjnius,kivy,python3,requests,urllib3,chardet,idna',
```

Comment:
**nitpick**: maybe we could keep this line below 80 characters?

```suggestion
                'sqlite3,libffi,openssl,pyjnius,kivy,python3,requests,urllib3,'
                'chardet,idna',
```

Reply:
Thanks addressed.
Honestly I thought about it, but I also thought why caring? We don't lint on this rule, we don't want to use black. I feel like if we want such things then we should commit for it, hence introduce the linting rule.
I'm in favour of it, specially black, but it has been discussed in the past and apparently we're not yet ready for it